core/pom.xml

```diff
@@ -258,6 +262,9 @@
                 <allow>javax[.]security[.]auth[.]DestroyFailedException</allow>
                 <!-- allow questionable Hadoop exceptions for mapreduce -->
                 <allow>org[.]apache[.]hadoop[.]mapred[.](FileAlreadyExistsException|InvalidJobConfException)</allow>
+                <!-- allow the following types from the visibility API -->
+                <allow>org[.]apache[.]accumulo[.]access[.]AccessExpression</allow>
+                <allow>org[.]apache[.]accumulo[.]access[.]IllegalAccessExpressionException</allow>
```

Comment:
It appears that you might not be throwing this Exception and are throwing other types instead.

Reply:
The reason for this is that org.apache.accumulo.core.security.VisibilityParseException now has a constructor that takes IllegalAccessExpressionException.